definitions/ext-service/summary_metrics.yml

```diff
@@ -8,20 +8,20 @@ responseTimeMs:
   unit: SECONDS
   queries:
     opentelemetry:
-      select: average(duration.ms) / 1000
+      select: average(duration.ms)
```

Comment:
the unit is `SECONDS`, why are we not dividing by 1000 here?

Reply:
I'm changing it. I just didn't check, since my default action is to change the summary where the implications are low